service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -57,6 +59,77 @@ public class MigrateCaseService {
     private final CourtService courtService;
     private final DocumentListService documentListService;
 
+    public Map<String, Object> rollbackCourtBundleMigration(CaseData caseData) {
+        List<Element<HearingCourtBundle>> newHearingCourtBundleList = new ArrayList<>();
+
+        newHearingCourtBundleList.addAll(caseData.getHearingDocuments().getCourtBundleListV2());
+        newHearingCourtBundleList.addAll(caseData.getHearingDocuments().getCourtBundleListLA());
+        newHearingCourtBundleList.addAll(caseData.getHearingDocuments().getCourtBundleListCTSC());
+
+        Map<String, Object> ret = new HashMap<>();
+        ret.put("courtBundleListV2", newHearingCourtBundleList);
+        ret.put("courtBundleListLA", null);
+        ret.put("courtBundleListCTSC", null);
+        return ret;
+    }
+
+    public void removeUnusedCourtBundleFields(CaseDetails caseDetails) {
```

Comment:
Could this method be moved inside `rollbackCourtBundleMigration` so we wouldn't miss it? Will need to change the method signature.